README.md

```diff
@@ -126,13 +126,15 @@ compares QuestDB to other open source time series databases spanning functionali
 
 Here are high-cardinality
 [Time Series Benchmark Suite](https://questdb.io/blog/2021/06/16/high-cardinality-time-series-data-performance/)
-results using the `cpu-only` use case with 6 to 16 workers on 32 CPUs and 64GB RAM:
+results using the `cpu-only` use case with 12 workers on 48 CPUs and 96GB RAM:
 
 <div align="center">
-    <img alt="A chart comparing the ingestion rate of QuestDB, InfluxDB and TimescaleDB." src=".github/readme-benchmark.png" width="600"/>
+    <img alt="A chart comparing the ingestion rate of QuestDB, InfluxDB and TimescaleDB." src=".github/questdb7.3.10-tsbs-benchmark.png" width="600"/>
   </a>
 </div>
 
+Competing TSDBs like InfluxDB and Timescale take a performance hit as the number of hosts grows. By contrast, not only does QuestDB soar above the competition at any host count, it maintains its high ingestion speed as cardinality increases.
+
```

Comment:
I would flip this around:

This benchmark measures the ingestion rate (rows/second) for different numbers of hosts 100, 1K, 100K and 10M hosts. The larger the number of hosts, the higher the dataset cardinality. As the number of hosts increases, QuestDB maintains its ingestion speed above 2.4M rows/s. By contrast, as the dataset cardinality increases, InfluxDB and TimescaleDB take a performance hit, with the ingestion rate degrading significantly.

Reply:
Nice one.